doc/news/changes.h

```diff
@@ -90,6 +90,13 @@ inconvenience this causes.
 
 
 <ol>
+  <li> New: There is now a function project_real_point_to_unit_point_on_face
```

Comment:
Write this (and the following function reference) as `Mapping::project_real_point_to_unit_point_on_face()` and doxygen will automatically cross-link it for you.